packages/web-components/fast-foundation/src/anchored-region/anchored-region.template.ts

```diff
@@ -10,7 +10,7 @@ export const anchoredRegionTemplate: (
     context,
     definition
 ) => ViewTemplate<AnchoredRegion> = (context, definition) => html`
-    <template class="${x => (x.initialLayoutComplete ? "loaded" : "")}">
+    <template class="${x => (x.regionVisible ? "loaded" : "")}">
```

Comment:
do we need the prefix of `region` here since this is on the region itself?

Reply:
There's already a 'visible' prop with a different meaning.

Reply:
went with "contentRendered" instead